assets/js/blocks/rating-filter/utils.ts

```diff
@@ -18,10 +23,28 @@ export const getActiveFilters = ( queryParamKey = 'filter_rating' ) => {
 	return parsedParams;
 };
 
+export function generateUniqueId() {
+	return Math.floor( Math.random() * Date.now() );
+}
+
+export const formatSlug = ( slug: string ) =>
+	slug
+		.trim()
+		.replace( /\s/g, '-' )
+		.replace( /_/g, '-' )
+		.replace( /-+/g, '-' )
+		.replace( /[^a-zA-Z0-9-]/g, '' );
```

Comment:
Given that _Filter by Stock_ will also need these utils. I wonder if we should extract them to a common file. What do you think?

Reply:
Sounds good! :100: